features/webxr-camera.yml

```diff
@@ -0,0 +1,10 @@
+name: WebXR Raw Camera Access
+description: Direct access to the camera allowing pose-synchronized camera images within a WebXR session.
```

Comment:
This is not a sentence—an active verb would really help here. What does this thing do? I suspect it's something like…

> The [mumble mumble] interface reads from …

But I have no idea what goes here and there's no XRCamera docs to look at.